src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/ImplementIDisposableCorrectly.cs

```diff
@@ -228,17 +228,9 @@ private void AnalyzeNamedTypeSymbol(SymbolAnalysisContext context)
                                 }
                             }
                         }
-                        else if (type.Interfaces.Contains(_disposableType))
-                        {
-                            // Reports violation, when type mentions IDisposable as implemented interface,
-                            // even when Dispose method is not implemented, but inherited from base type
-                            // For example see unit test:
-                            // CSharp_CA1063_IDisposableReimplementation_Diagnostic_ReImplementingIDisposableWithNoDisposeMethod
-                            CheckIDisposableReimplementationRule(type, context, implementsDisposableInBaseType);
-                        }
                     }
 
-                    if (implementsDisposableInBaseType)
+                    if (implementsDisposableInBaseType && FindInheritedDisposeBoolMethod(type) != null)
```

Comment:
This only restricts calls to _CheckFinalizeOverrideRule_. CheckFinalizeImplementationRule is called from AnalyzeOperationBlock (though apparently it's not implemented yet anyway).
Regarding the second point, that diagnostic exists (see call to CheckProvideDisposeRule) and is reported only in the type that actually implements the Dispose() method. We don't want to direct every type with an IDisposable ancestor to provide a new, virtual Dispose(bool) implementation.